src/_core/lodash-utils.js

```diff
@@ -179,4 +179,18 @@ export var buildIsMethods = (namespace, target) => {
 lodashUtils.buildIsMethods = buildIsMethods;
 
 
+/**
+ * Build `before` and `after` methods for moment
+ *
+ * @method buildInclusiveCompare
+ * @param {String} method: either 'isBefore' or 'isAfter'
+ * @param {Object} target: namespace to overload methods on
+ * @return {Function}
+ */
+export var buildInclusiveCompare = (method, target) => {
```

Comment:
I don't think we need to export this helper method- maybe prefix with underscore as well